objective-c/HelloWorld/HelloWorld/AppDelegate.h

```diff
@@ -0,0 +1,17 @@
+//
+//  AppDelegate.h
+//  HelloWorld
+//
+//  Created by Michael Lumish on 6/11/15.
+//  Copyright (c) 2015 Google. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface AppDelegate : UIResponder <UIApplicationDelegate>
+
+@property (strong, nonatomic) UIWindow *window;
+
+
```

Comment:
Nit: you can remove these extra newlines, for consistency with the other example.